Configuration/TSconfig/ContentElementWizard.tsconfig

```diff
@@ -1,18 +1,18 @@
 mod.wizards.newContentElement.wizardItems.plugins {
   elements {
     weather2_current {
-      iconIdentifier = ext-weather2-table-currentweather
+      iconIdentifier = ext-weather2
       title = LLL:EXT:weather2/Resources/Private/Language/locallang_db.xlf:plugin.currentweather.title
-      description = LLL:EXT:weather2/Resources/Private/Language/locallang_db.xlf:plugin.currentweather.title
+      description = LLL:EXT:weather2/Resources/Private/Language/locallang_db.xlf:plugin.currentweather.descriptiom
       tt_content_defValues {
         CType = list
         list_type = weather2_currentweather
       }
     }
     weather2_alert {
-      iconIdentifier = ext-weather2-table-weatheralert
+      iconIdentifier = ext-weather2
```

Comment:
Now, two different plugins have same icon

Reply:
It's the generic extension weather icon. And in the old version also it is a generic icon.

Reply:
I updated the icons now.